src/model/error.rs

```diff
@@ -111,28 +111,30 @@ impl fmt::Display for Minimum {
 /// use serenity::prelude::*;
 /// use serenity::Error;
 ///
-/// # #[cfg(feature = "http")]
-/// # async fn run() -> Result<(), Box<dyn std::error::Error>> {
-/// # let http: serenity::http::Http = unimplemented!();
-/// # let guild_id: GuildId = unimplemented!();
-/// # let user: User = unimplemented!();
+/// const BAN_DURATION: u32 = 8 * 24 * 60 * 60;
 ///
-/// match guild_id.ban(&http, user.id, 8, Some("No unbanning people!")).await {
-///     Ok(()) => {
-///         // Ban successful.
-///     },
-///     Err(Error::Model(ModelError::TooLarge {
-///         value, ..
-///     })) => {
-///         println!("Failed deleting {value} days' worth of messages");
-///     },
-///     Err(why) => {
-///         println!("Unexpected error: {why:?}");
-///     },
-/// }
+/// # struct Handler;
 ///
-/// # Ok(())
-/// # }
+/// # #[cfg(feature = "gateway")]
+/// #[serenity::async_trait]
+/// impl EventHandler for Handler {
+///     async fn guild_ban_removal(&self, ctx: Context, guild_id: GuildId, user: User) {
+///         match guild_id.ban(&ctx.http, user.id, BAN_DURATION, Some("No unbanning people!")).await
+///         {
+///             Ok(()) => {
+///                 // Ban successful.
+///             },
+///             Err(Error::Model(ModelError::TooLarge {
+///                 value, ..
+///             })) => {
+///                 println!("Failed deleting {value} days' worth of messages");
+///             },
+///             Err(why) => {
+///                 println!("Unexpected error: {why:?}");
+///             },
+///         }
+///     }
+/// }
```

Comment:
Why rework this test to rely on gateway again?

Reply:
I used GitHub’s rebase

Reply:
Should be resolved